dashboard-new/src/app/(authenticated)/workflow/[...workflow]/components/Details.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+import { Listbox } from '@headlessui/react'
+import { TagIcon } from '@heroicons/react/24/outline'
+import { WfSpecId } from 'littlehorse-client/dist/proto/object_id'
+import { WfSpec } from 'littlehorse-client/dist/proto/wf_spec'
+import React, { FC, useCallback, useState } from 'react'
+import { getWfSpecVersions } from '../getVersions'
+import { useWhoAmI } from '@/contexts/WhoAmIContext'
+
+type DetailsProps = Pick<WfSpec, 'id' | 'status'>
+
+const statusColors: { [key in WfSpec['status']]: string } = {
+  ARCHIVED: 'bg-gray-200',
+  ACTIVE: 'bg-blue-200',
+  TERMINATING: 'bg-yellow-200',
+  UNRECOGNIZED: 'bg-red-200',
+}
+
+export const Details: FC<DetailsProps> = ({ id, status }) => {
+  return (
+    <div className="mb-4">
+      <span>Workflow</span>
```

Comment:
"Workflow" is ambiguous. Do you mean `WfSpec` or `WfRun`?

Reply:
In this context is WfSpec, I'm thinking on simplify the names,

WfSpec = Workflow
WfRun = Workflow Run

Or humanize

WfSpec = Workflow Spec
WfRun = Workflow Run